### PR TITLE
RA-1861: Clear and disable relationships for unknown patients.

### PR DESCRIPTION
**Pull request by icrc-toliveira**
RA-1861 - The relationships field is now grey-lined / disabled

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/registerPatient.js [hunk 1]
```diff
@@ -297,8 +297,25 @@ jq(function() {
...
+
+            $("#personRelationshipField p input").addClass( "disabled");
+            $("#personRelationshipField p input").prop( "disabled", true);
+            $("#personRelationshipField p input").val(null);
+
+            $("#personRelationshipField p a").each(function(i, button) {
+                if ((i % 2) != 0){
+                    button.click();
+                }
+            });
```
Comment (mks-d) [inline omod/src/main/webapp/resources/scripts/registerPatient.js:315]: @icrc-toliveira this clicks on the second button each time right, the "minus" one?

If so why not do that first? As in:
1. Click on all minus buttons, then
2. Disable and clear the last remaining relationship row.

Comment (icrc-toliveira) [reply omod/src/main/webapp/resources/scripts/registerPatient.js]: Ok, 
Done

File: omod/src/main/webapp/resources/scripts/registerPatient.js [hunk 2]
```diff
@@ -338,16 +357,25 @@ jq(function() {
...
         }
         else {
+            // Disable "HSU related" dynamic field
+            $("#personRelationshipField h3").removeClass("disabled");
+
+            $("#personRelationshipField p select").removeClass("disabled");
+            $("#personRelationshipField p select").prop( "disabled", false);
+
+            $("#personRelationshipField p input").removeClass( "disabled");
+            $("#personRelationshipField p input").prop( "disabled", false);
```
[the remark this answers is not in the document]
Comment (sherrif10) [reply omod/src/main/webapp/resources/scripts/registerPatient.js:367]: Hello @mks-d , could you help me with the meaning of HSU  in openmrs context am not understanding this word neither online thanks

**Comment by gracepotma** [comment]
@mks-d  thanks for reviewing this! Is it ready to be merged?

**Review by mks-d — APPROVED** [review]

**Pull request merged**